nano/node/json_handler.cpp

```diff
@@ -1950,11 +1950,11 @@ void nano::json_handler::delegators ()
 		auto transaction (node.store.tx_begin_read ());
 		for (auto i (node.store.latest_begin (transaction)), n (node.store.latest_end ()); i != n; ++i)
 		{
-			nano::account_info const & info (i->second);
-			if (info.representative == account)
+			auto state (node.ledger.account_state (transaction, nano::account_info (i->second)));
```

Comment:
This is going to make the delegators RPC slow again. The 1.1M+ point reads for blocks can take a couple minutes on some systems. I assumed we would be combining the `head` block into the `account_info` and removing duplicate fields so that we could get info about both in 1 I/O read (and in this case still scan a range quickly (100-150ms).